frontend/src/pages/Dashboard.tsx

```diff
@@ -28,6 +31,14 @@ const rows = [
 ];
 
 export default function Dashboard() {
+  const auth = useAuth();
+
+  // TODO: Remove this useEffect and replace it with a real API call using react-router data loaders
+  useEffect(() => {
+    pingBackend(auth).then(console.log);
+  });
```

Comment:
We can simulate that by simply returning the mocked data in the data loader function

Reply:
will fix in a seperate PR